patient-view-be/src/main/java/spvproject/patientviewbe/controller/PainAssessmentController.java

```diff
@@ -0,0 +1,23 @@
+package spvproject.patientviewbe.controller;
+
+
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import org.springframework.web.bind.annotation.RestController;
+
+import spvproject.patientviewbe.model.PainAssessmentModel;
+
+@RestController
+@RequestMapping("/painAssessment")
```

Comment:
REST API routes when containing multiple words should be split by `-` and all lower case